markdown/m2h/handlers/index.ts

```diff
@@ -27,35 +27,61 @@ function getL10nCardMap(locale = DEFAULT_LOCALE) {
 
   Object.keys(listMsgObj).forEach((msgName) => {
     l10nCardMap.set(
-      listMsgObj[msgName]["msgstr"][0],
-      listMsgObj[msgName]["msgid"]
+      listMsgObj[msgName]["msgid"],
+      listMsgObj[msgName]["msgstr"][0]
     );
   });
   return l10nCardMap;
 }
 
-function getNotecardType(node, locale) {
+interface NotecardType {
+  type: string;
+  isGFM: boolean;
+  magicKeyword: string;
+}
+
+function getNotecardType(node: any, locale: string): NotecardType | null {
+  const types = ["note", "warning", "callout"];
```

Comment:
I feel like we shouldn't support callout here, if it isn't part of the GFM noteblocks proposal (and therefore not rendered by GitHub):

> [!CALLOUT]
> Callout

> [!IMPORTANT]  
> Crucial information necessary for users to succeed.

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

@mdn/core-yari-content What do you think?

Reply:
Note: if we remove support for callouts in the GFM syntax, we'd have to special-case it for the original syntax.  If the decision is not to support callouts in GFM syntax, then we should overhaul all existing callouts too.